FundacionParaguaya/MentorApp
# Tip changes

**Describe the changes**
Tip functionality update. Now the tip is part of the sticky footer and shows/hides in accordance with the requirements. You can interact with the app while the tip is showing.

**To Test**
Test skipped questions screen and overview screen. See that the tip appears/hides appropriately, has the correct information and is replaced by the continue button when it is hidden. Test some other screens with continue button to verify that the functionality of the button and the sticky footer has not been altered.
